### ISSUE TITLE
Loss_weight_Problem

### ISSUE THREAD

**Issue by Dali936** [IsshikiHugh/HSMR#49]
Hi Yan,
    I’d like to confirm whether the main difference between the 4DHuman dataset and your dataset lies in the annotations — specifically, the use of SMPL parameters versus SKEL parameters. I also noticed that when the network converges, the primary loss (kp3d) is around 0.05, while the loss on the fitted SKEL parameters is only about 0.001. Have you conducted any ablation studies to compare training with and without SKEL parameters?

<img width="877" height="222" alt="Image" src="https://github.com/user-attachments/assets/18f12cda-7ed3-438b-90ff-b1a949f553f6" />

**Comment by Dali936** [comment]
> Hi [@Dali936](https://github.com/Dali936) ,
> 
> That's an interesting point! But actually no. We don't do much experiments on the hyperparameters. However, regarding the mission of parameters, I think the ablation study about the rotation representations can tell something. But I think some experiments on this could also be interesting. You can do this ablation on ViT-B version of HSMR.
> 
> Best, Yan

Thanks, i will let you know if i have further analysis
